.codegen.json

```diff
@@ -11,10 +11,10 @@
     "toolchain": {
         "required": ["go"],
         "post_generate": [
-            "go run ./bundle/internal/bundle/schema/main.go ./bundle/schema/docs/bundle_descriptions.json",
+            "go run ./bundle/internal/schema/*.go ./cmd/bundle/_generated/jsonschema.json",
             "echo 'bundle/internal/tf/schema/\\*.go linguist-generated=true' >> ./.gitattributes",
             "echo 'go.sum linguist-generated=true' >> ./.gitattributes",
-            "echo 'bundle/schema/docs/bundle_descriptions.json linguist-generated=true' >> ./.gitattributes"
+            "echo 'cmd/bundle/_generated linguist-generated=true' >> ./.gitattributes"
```

Comment:
Could you store this in `bundle/internal/schema` somewhere to keep it local to the bundle directory?

Reply:
So:
1. Embedded files have to be inside the go package. `//go:embed ../a/b/c` is not allowed.
2. We cannot import objects from a package inside `internal`

To still keep the generated file inside the bundle package, I created `bundle/generated`.

Reply:
We reference the file like this now in the `bundle schema` command:
```
_, err := cmd.OutOrStdout().Write(generated.BundleSchema)
```